### PR TITLE
Remove optimize_lookup setting.

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

File: bench/runbench [hunk 1]
```diff
@@ -16,5 +16,5 @@ filename=bench/log/`date +%F_%T`.txt
 rm -rf venv-bench
 virtualenv venv-bench
 . venv-bench/bin/activate
-CHEETAH_C_EXT=true pip install .
+pip install .
```
Comment (bukzor) [inline bench/runbench]: why this change?

Comment (asottile) [inline bench/runbench]: yakshave, I forgot to remove this when we removed that setting
